ocis-pkg/shared/errors.go

```diff
@@ -69,3 +69,19 @@ func MissingAdminUserID(service string) error {
 		"the config/corresponding environment variable).",
 		service, defaults.BaseConfigPath())
 }
+
+func MissingServiceAccountID(service string) error {
```

Comment:
exported func `MissingServiceAccountID` should have a comment or be unexported

Reply:
this is a backport and we should not add something that is not in master